src/main/java/io/codelex/arrays/practice/My_TicTacToe.java

```diff
@@ -0,0 +1,174 @@
+package io.codelex.arrays.practice;
+
+import java.util.Scanner;
+
+public class My_TicTacToe {
+
+    private final static String playerX = "X";
+    private final static String playerO = "O";
+    private final static String[][] table = new String[3][3];
+
+    public static void main(String[] args) {
```

Comment:
Šajā metodē izskatās tā kā par daudz vienāda koda. Jo faktiski tā pirmā IF čupa ir identiska ar otro čupu, atšķirās tikai tas vai viņš pārbauda vai sakrīt ar playerX vai player0. To kopīgo vajadzētu iznest metodē, kurai padod String player un viņa atgriež boolean vērtību vai ir uzvarējis vai nav. Tad arī nebūtu jātkārto burtiski kādas 16x tas You win un break.